Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1537,6 +1537,9 @@ NOTE: Restart is required for the setting to take effect.</value>
   <data name="hourglass" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>Icons\Fugue\hourglass.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="skull_mad" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>Icons\Fugue\skull-mad.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

Comment:
should it be here? Just wondering because I think that was for the spectacular fail which should be removed (if not already) so I'm not sure why it's being added here.

Reply:
IIRC this PR predates that removal. Git may have automatically resolved conflicts the wrong way here